js/app.js

```diff
@@ -1 +1,202 @@
+  console.log(playerHand, dealerHand)
+  console.log(cardDeck)
+}
+
+
+function renderCards() {
+  playerInfo.innerHTML = ""
+  dealerInfo.innerHTML = ""
+  for(i= 0; i< playerHand.length; i++) {
+    let card = document.createElement("div")
+    card.className = `card xlarge ${playerHand[i]}`
+    playerInfo.appendChild(card)
+  }
+  for(i = 0; i < dealerHand.length; i++) {
+    let card = document.createElement("div")
+    card.className = `card xlarge ${dealerHand[i]}`
+    dealerInfo.appendChild(card)
+  }
+  hitbtn.style.visibility = "visible"
+  standbtn.style.visibility = "visible"
+}
+
+
+function cardValues(card) {
+  let value
+  
+  if (ace.includes(card)) {
+    value = 11
+  }
+  if (pictureCard.includes(card)) {
+    value = 10
+  }
+  if (nineSuits.includes(card)) {
+    value = 9
+  }
+  if (eightSuits.includes(card)) {
+    value = 8
+  }
+  if (sevenSuits.includes(card)) {
+    value = 7
+  }
+  if (sixSuits.includes(card)) {
+    value = 6
+  }
+  if (fiveSuits.includes(card)) {
+    value = 5
+  }
+  if (fourSuits.includes(card)) {
+    value = 4
+  }
+  if (threeSuits.includes(card)) {
+    value = 3
+  }
+  if (twoSuits.includes(card)) {
+    value = 2
+  }
+  return value 
+}
+
+function countHands(hand) {
+  let handTotal = 0
+
+  for (let i = 0; i< hand.length; i++) {
+    handTotal = handTotal + cardValues(hand[i])
+}
+  return handTotal
+}
+
+function hit() {
+  playerHand.push(cardDeck.pop())
+  renderCards()
+  playerTotal.textContent = countHands(playerHand)
+  
+  if(countHands(playerHand) > 21) {
+    statusMsg.textContent = "Player BUSTS! Dealer Wins"
+    hitbtn.style.visibility = "hidden"
+    standbtn.style.visibility = "hidden"
+  }
+}
+
+function stand() {
+  while(countHands(dealerHand) < 17) {
+    dealerHand.push(cardDeck.pop())
+    renderCards()
+    dealerTotal.textContent = countHands(dealerHand)
+}
+  hitbtn.style.visibility = "hidden"
+  standbtn.style.visibility = "hidden"
+
+if (countHands(dealerHand) > 21) {
+  statusMsg.textContent = "Dealer Busts! Player WINS"
+}
+else if(countHands(dealerHand) > countHands(playerHand)) {
+  statusMsg.textContent = "Dealer WINS!"
+}
+else if(countHands(dealerHand) === countHands(playerHand)) {
+  statusMsg.textContent = "It's a PUSH!"
+}
+else {
+  statusMsg.textContent = "Player WINS!"
+}
+}
+
+function checkBlackjack(pHand, dHand) {
+  if(pHand === 21 && dHand === 21){
+    statusMsg.textContent = "Both have BlackJack! Its a PUSH"
+    hitbtn.style.visibility = "hidden"
+    standbtn.style.visibility = "hidden"
+  }
+else if (pHand === 21) {
+  statusMsg.textContent = "BLACKJACK! PLAYER WINS"
+  hitbtn.style.visibility = "hidden"
+  standbtn.style.visibility = "hidden"
+}
+else if(dHand === 21) {
+  statusMsg.textContent = "Dealer has BlackJack! Dealer WINS"
+  hitbtn.style.visibility = "hidden"
+  standbtn.style.visibility = "hidden"
+}
+  playerTotal.textContent = countHands(playerHand)
+  dealerTotal.textContent = countHands(dealerHand)
+}
```

Comment:
Your indentation gets a wonkey bit in these last couple functions, the rest of your code looks great though. take a couple minutes to fix this.